README.md

```diff
@@ -34,6 +34,14 @@ Questions? Comments? Start by perusing our [GitHub discussions](https://github.c
 
 Please see [the documentation](https://nautobot.readthedocs.io/en/stable/installation/) for instructions on installing Nautobot.
 
+## Application Stack
+
+Below is a simplified overview of the Nautobot application stack for reference:
+
+![Application stack diagram](/docs/media/nautobot_application_stack_high_level.png "Application stack diagram")
+
+Find more details [here](nautobot/docs/index.md).
```

Comment:
Since README.md gets published to PyPI, these both probably need to be absolute links to ReadTheDocs URLs rather than relative file links, so that they won't be broken links on PyPI.